### ISSUE TITLE
why

### ISSUE THREAD

**Issue by NextinMono** [samuelmarina/is-even#158]
just why

**Comment by jaxparrow07** [comment]
This helps the community.. lol.. You don't understand the pain of programmers..

**Comment by arpan-lol** [comment]
why not

**Comment by sudeepgumaste** [comment]
because...

**Comment by Hocuri** [comment]
Yeah, this is really useful, I use it everytime I want to know whether a number is even

**Comment by benben-6** [comment]
> just why

why the fuck not???
Many Young kids have struggled figuring if a number is even, this project was made for those poor, poor children!

**Comment by kyuridenamida** [comment]
The essence to comprehend the "why" is that only human beings can lead A.I. in the right direction.  We, programmers, had been failing to tackle the even number judgment problem explicitly as a standalone problem because it was too obvious. 

This repository will be the important first step to realize the future that coding A.I. technology such as Github Copilot takes over all the coding tasks we have.

**Comment by arpan-lol** [comment]
> The essence to comprehend the "why" is that only human beings can lead A.I. in the right direction. We, programmers, had been failing to tackle the even number judgment problem explicitly as a standalone problem because it was too obvious.
> 
> This repository will be the important first step to realize the future that coding A.I. technology such as Github Copilot takes over all the coding tasks we have.

Gthub Copilot **can't** and **won't** take over _all_ of the coding tasks. Please don't stereotype lol

**Comment by sudeepgumaste** [comment]
Tell me why..

**Comment by niltonfrederico** [comment]
> Tell me why..

Ain't not but an odd ache
